Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -84,7 +84,7 @@ SearchBarSectionControllerDelegate {
     // MARK: SearchBarSectionControllerDelegate
 
     func didChangeSelection(sectionController: SearchBarSectionController, query: String) {
-        filter(query: query.trimmingCharacters(in: .whitespacesAndNewlines), animated: true)
+        filter(query: query, animated: true)
```

Comment:
Removed as this is redundant; it already happens in [`filtered()`](https://github.com/rnystrom/GitHawk/blob/master/Classes/Utility/Filterable.swift#L17).